docs/tutorial/03-fungible-tokens.mdx

```diff
@@ -28,13 +28,39 @@ In this tutorial, we're going to deploy, store, and transfer fungible tokens.
 Some of the most popular contract classes on blockchains today are fungible tokens.
 These contracts create homogeneous tokens that can be transferred to other users and spent as currency (e.g., ERC-20 on Ethereum).
 
-Usually, a central ledger keeps track of a user's token balances. With Cadence, we use a new resource paradigm to implement fungible tokens and avoid using a central ledger.
+In traditional software and smart contracts, balances for each user are tracked by a central ledger, such as a dictionary:
+```cadence
+contract LedgerToken {
+    // Tracks every user's balance
+    pub let balances: {Address: UFix64}
```

Comment:
I know this is "don't do this code" but this dictionary is publicly mutatable. Maybe  make it`access(contract) let` ?

Reply:
Good point. I'll change that